### ISSUE TITLE
EntityFramework: Unable to cast object of type 'System.UInt64' to type 'System.Int32'

### ISSUE THREAD

**Issue by caleblloyd** [mysql-net/MySqlConnector#42]
```
System.InvalidCastException: System.InvalidCastException: Unable to cast object of type 'System.UInt64' to type 'System.Int32'..
Stack Trace:
at MySql.Data.MySqlClient.MySqlDataReader.GetInt32(Int32 ordinal)
at lambda_method(Closure , DbDataReader )
at Microsoft.EntityFrameworkCore.Storage.Internal.TypedRelationalValueBufferFactory.Create(DbDataReader dataReader)
at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.ConsumeResultSetWithPropagation(Int32 commandIndex, DbDataReader reader)
at Microsoft.EntityFrameworkCore.Update.Internal.MySqlModificationCommandBatch.Consume(DbDataReader reader)
at Microsoft.EntityFrameworkCore.Update.Internal.MySqlModificationCommandBatch.Execute(IRelationalConnection connection)
at Microsoft.EntityFrameworkCore.Update.Internal.MySqlBatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IReadOnlyList`1 entriesToSave)
at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)
```

**Comment by caleblloyd** [comment]
Solution is to cast unsigned types, PR #43

**Comment by bgrainger** [comment]
Fixed in 626e9be918ab156e04abd7d3277ae8bf0948ff47 and [0.1.0-alpha16](https://www.nuget.org/packages/MySqlConnector/0.1.0-alpha16).

**Comment by caleblloyd** [comment]
Thanks @bgrainger!
